src/lightkurve/lightcurve.py

```diff
@@ -3140,7 +3204,143 @@ def plot_river(self, **kwargs):
             period=self.period, epoch_time=self.epoch_time, **kwargs
         )
         return ax
+    
+
+    def bin(
+        self,
+        time_bin_size=None,
+        time_bin_start=None,
+        time_bin_end=None,
+        aggregate_func=None,
+        bins=None,
+        n_bins=None,
+
+    ):
+        """Bins a FoldedLightCurve in equally-spaced bins in phase.
+        Binning always occurs in time units (not normalized phase units)
+
+        If the original light curve contains flux uncertainties (``flux_err``),
+        the binned lightcurve will report the root-mean-square error.
+        If no uncertainties are included, the binned curve will return the
+        standard deviation of the data.
 
+        Parameters
+        ----------
+        time_bin_size : `~astropy.units.Quantity` or `~astropy.time.TimeDelta`, optional
```

Comment:
The docstring for time_bin_size should be updated to explain the semantics for normalized phase (that it can accept either time or normalized phase).